docs/guidelines/content-guidelines/04-ui-elements.md

```diff
@@ -0,0 +1,110 @@
+---
+title: UI elements
+---
+
+Contrary to the regular technical documentation, UI elements follow different guidelines and best practices. Due to the limited space you can use to convey a message, your UI text must be as precise and concise as possible. Use the following general guidelines to design and write content for UI elements:
+
+- Use active voice and action-oriented language.
+- Use title case for short one-line texts, such as labels or headings, and sentence case for messages and explanations, unless specified otherwise. (?)
+- Use plural for the object names, as users usually interact with multiple objects within the program, for example:
+
+  ✅ Service Instances <br>
+  ⛔️ Service Instance
+
+- Use plain, natural language. Even though some verbs may be similar in meaning, they may sound more or less natural to the reader. For example, from these pairs of verbs, the first one sounds more natural:
+
+  ✅ Choose <br>
+  ⛔️ Select
+
+  ✅ Edit <br>
+  ⛔️ Modify
+
+- Use the three-letter currency code instead of the currency symbol. This prevents ambiguity when multiple currencies share the same symbol.
+
+  ✅ USD <br>
+  ⛔️ $
+
+
+## Labels
+
+Labels are one-line texts that appear in various types of UI elements, such as buttons, titles and headings, forms, and drop-down menus. When designing labels for the UI elements in the Kyma project, make sure to:
+
+- Use title case for titles and headings.
+- Use sentence case in forms, drop-down menus, and button labels.
+- Omit punctuation to avoid "word pollution".
+
+### Buttons
+
+When designing a button label, you must be as straightforward and concise as possible, since you have very limited space to convey the message. Moreover, buttons are used to invite a user to perform an action. Make them as inviting and intuitive as possible. To do so:
+
+- Use action verbs that explain what will happen when a user presses the button, without requiring them to read any supporting text. Depending on your needs, you can also use plain adverb. Alternatively, add a noun to indicate what a verb or an adverb refers to. Do **not** use articles and punctuation. They only add unnecessary "word pollution" to your button label.
+
+  ✅ Save <br>
+  ✅ Next <br>
+  ✅ Save draft <br>
+  ✅ Next step <br>
+  ⛔️ Save a draft
+
+- Avoid labels that are too generic, such as `Yes` or `No`. They are not intuitive and require the user to read the supporting text.  
+- Use sentence case as it reads more natural and friendly to the users, which makes them more willing to press the button. Title case doesn't read as easily as sentence case and looks more unnatural.
```

Comment:
That's in conflict with the [UA Style Guide](https://help.sap.com/docs/SNG_FOR_UA/e33c591ae4494a659a3f5f983c9d1161/79ce282052d14cc38e6a203a1fc277c4.html?state=DRAFT&locale=en-US&q=ui%20elements) and the [Fiori UI Guide](https://experience.sap.com/internal/fiori-design-web/ui-text-guidelines-for-sap-fiori/#formatting) 😐  

> Use title case for labels, titles, and similar short texts on the user interface. Here are some examples: menu items; text on buttons and icons;...

> Unless otherwise specified for individual UI elements, use title case for all SAP Fiori user interface short texts (labels, headings, value help texts, and so on), and sentence case for all messages and explanations.